### PR TITLE
Fix: Followed users remove from suggestion

### PR DESCRIPTION
**Pull request by pritam-kr**

### CODE REVIEW

File: src/Pages/Bookmark/Bookmark.jsx [hunk 1]
```diff
@@ -28,14 +27,24 @@ const Bookmark = () => {
   }, [token, dispatch]);
 
   //Getting bookmark posts by filtering from all post
-  const bookmarkPost = bookmarks?.map((eachId) => allPost?.find((eachPost) => eachPost._id=== eachId))
+  const bookmarkPost = bookmarks?.map((eachId) =>
+    allPost?.find((eachPost) => eachPost._id === eachId)
+  );
 
   // filter user on the basis of user name for follow/ unfollow
```
Comment (varnita619) [inline src/Pages/Bookmark/Bookmark.jsx:34]: good use of comments

**Review by varnita619 — APPROVED** [review]
UI is clean, function working properly
Code is clean and modular
Ready to merge

**Review by AbhishekkGautam — APPROVED** [review]
Everything looks good. Good to merge

**Review by samyakshah3008 — APPROVED** [review]
Clean code and UI 
Good for approval

**Pull request merged**